terraform/dev/gke.tf

```diff
@@ -0,0 +1,51 @@
+module "gke" {
+  source     = "terraform-google-modules/kubernetes-engine/google//modules/private-cluster"
```

Comment:
I see "//" in the path. Is that intentional?

Reply:
yup. Its intentional.

```
A special double-slash syntax is interpreted by Terraform to indicate that the remaining path after that point is a sub-directory within the package
```